src/main.ts

```diff
@@ -16,9 +16,17 @@ export async function run(): Promise<void> {
     const { summary } = await analyze(path)
 
     const rate = summary.lines.hit / summary.lines.found
+
```

Comment:
Good idea to property set the output.
I think we need to set more outputs: `coverage` - the total coverage,  `diff` the diff coverage(between total coverages), `patch ` - the patch coverage.